src/vellum/client/__init__.py

```diff
@@ -116,23 +118,34 @@ class Vellum:
     from vellum import Vellum
 
     client = Vellum(
-        api_key="YOUR_API_KEY",
+        workspace_api_key="YOUR_WORKSPACE_API_KEY",
+        environment_api_key="YOUR_ENVIRONMENT_API_KEY",
     )
     """
 
     def __init__(
         self,
         *,
         environment: VellumEnvironment = VellumEnvironment.PRODUCTION,
-        api_key: str,
+        workspace_api_key: typing.Optional[str] = os.getenv("VELLUM_API_KEY"),
+        environment_api_key: typing.Optional[str] = os.getenv("VELLUM_API_KEY"),
         timeout: typing.Optional[float] = None,
         follow_redirects: typing.Optional[bool] = True,
         httpx_client: typing.Optional[httpx.Client] = None,
     ):
         _defaulted_timeout = timeout if timeout is not None else None if httpx_client is None else None
+        if workspace_api_key is None:
+            raise ApiError(
+                body="The client must be instantiated be either passing in workspace_api_key or setting VELLUM_API_KEY"
+            )
+        if environment_api_key is None:
+            raise ApiError(
+                body="The client must be instantiated be either passing in environment_api_key or setting VELLUM_API_KEY"
+            )
```

Comment:
Treats them both as required - intentional? the typing of both being optional makes me think we want to ensure that at least one is not null, not both